docs/Tutorial.md

```diff
@@ -89,6 +89,28 @@ body {
 }
 ```
 
+Lastly, add the `Roboto` font to the `index.html` file:
```

Comment:
I'd put the custom css and the font after the initial screenshot, as it's strictly speaking not necessary to get the welcome screen (and it is more readable like that)